index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" media="screen" href="css/style.css">
+
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1>"hello world, we are software developers!"</h1>
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
```

Comment:
Please try to delete the comments that you don't need or not be so necessary, it will make you clean your code.

Reply:
- [ ] [OPTIONAL] As I mentioned above you sounded very polite and your explanation is good :+1: : . you can try using a couple of emojis and/or gifs in the review to make the code review more human-friendly. :smile_cat: